config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/BasicBeeStats-AAAAAAAAAAAAAAAAAAAIow==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "There are a couple of basic bees that have good stats you will want to breed into your breeding stock.\n\nCave Dwelling - Rocky - This lets you put the bee housing underneath or inside. Handy for putting a chest on top for products.\n\nNocturnal - Rocky - This keeps your bees working all night long, and in the dark.\n\nRain Tolerant - Water - Keep producing even when it is raining.\n\nHigh Fertility - Wintry - For breeding purposes, you want the maximum number of drones produced.\n\nReminder - Specialty products from bees require Jubilant state. That means the bee is in its §lNATURAL§r preferred climate. So it must not be in a biome it can tolerate due to temp/humidity improvements. For example, a Salt bee needs Warm/Normal climate. Even if it has +- humidity and temp, it will not produce its Salt comb unless it is in a Warm/Normal climate. Use Alveary upgrades to make sure it is in its preferred temp/humidity."
+      "desc:8": "There are a couple of basic bees that have good stats yow want to breed into your breeding stock.\n\nCave Dwelling - Rocky - This lets you put the bee housing underneath or inside. Handy for putting a chest on top for products.\n\nNocturnal - Rocky - This keeps your bees working all night long, and in the dark.\n\nRain Tolerant - Water - Keep producing even when it is raining.\n\nHigh Fertility - Wintry - For breeding purposes, you want the maximum number of drones produced.\n\nKeep in mind that specialty products from bees require the Jubilant state. \nTo be in that state, a bee needs to be in its natural§r preferred climate - it must not be in a biome it can tolerate due its traits.\n\nFor example, a Salt bee needs Warm/Normal climate. Even if it has +- humidity and temp, it will not produce its Salt comb unless it is in a Warm/Normal climate. Use Alveary upgrades to make sure it is in its preferred temp/humidity."
```

Comment:
"you will" was better than "yow" ;)

the jubilant explanation wasnt great but now its worse, also what is §r for? Maybe:

"To be in this state, a bee needs have its naturally preferred humidity and temperature. Just being able to work due to tolerance traits is not enough."

Reply:
Thank you!
The §r is a leftover from the in-game editing.
I applied your version.